### PR TITLE
Set the reset timestamp for a wrapped counter 1ms earlier, to avoid start_time==end_time errors.

### LINKED ISSUE #185: No data is available in the monitoring
Hi ,
I have some issues with the latest stackdriver agent.

*   Version : both 6.0.2-1.focal & 6.0.3-1.focal
*   Operating system / distribution: Ubuntu 20.04 LTS

## Expected behavior

Monitor the Disk utilization to alert when disks are nearly full.

## Actual behavior
No data about Disk Utilization are pushed to the Monitoring system.
I have it on 2 Ubuntu 20.04 fresh install, but I tried to install 6.0.3-1.focal on an Ubuntu 18.04 and it's working fine. 

I have this kind of logs every minute:
```
juin 18 07:58:44 gce_instance collectd[13149]: write_gcm: can not take infinite value
juin 18 07:58:44 gce_instance collectd[13149]: write_gcm: wg_typed_value_create_from_value_t_inline failed for swap/percent/value! Continuing.
juin 18 07:58:44 gce_instance collectd[13149]: write_gcm: Unsuccessful HTTP request 400: {
                                                   "error": {
                                                     "code": 400,
                                                     "message": "Field timeSeries[0].points[0].interval.start_time had an invalid value of \"2020-06-17T22:56:44.709265-07:00\": Th>
                                                     "status": "INVALID_ARGUMENT"
                                                   }
                                                 }
juin 18 07:58:44 gce_instance collectd[13149]: write_gcm: Error talking to the endpoint.
juin 18 07:58:44 gce_instance collectd[13149]: write_gcm: wg_transmit_unique_segment failed.
juin 18 07:58:44 gce_instance collectd[13149]: write_gcm: wg_transmit_unique_segments failed. Flushing.
```

## Steps to reproduce
*   Create GCE Vm Ubuntu 20.04 LTS minimal
*   Install and launch Stackdriver agent with official method: https://cloud.google.com/logging/docs/agent/installation
*   On https://console.cloud.google.com/monitoring/alerting/policies/
Add an alerting policy with:
- Resource type: GCE VM Instance
- Metrics: Disk Utilization
- Filters:
   - device: sda1 or your device if not sda1
   - state: used
   - instance_id: your instance id
- A numeric value un Threshold field

--> Message in the monitoring screen : No data is available for the selected time frame.

### PR DESCRIPTION
**Pull request by igorpeshansky**
Fixes #185.

### CODE REVIEW

File: src/write_gcm.c [hunk 1]
```diff
@@ -3861,7 +3861,11 @@ static int wg_rebase_item(
 
   // If any counter wrapped, everybody resets.
```
Comment (jkschulz) [inline src/write_gcm.c]: nit: the comments on this and the preceding block seem flipped. I'd make the one above (3851-2) something like

> Check to see if any counters have decreased from their last value (indicating a wrap or a reset). If one has, we'll reset everyone.

and bring the existing one down to this block.

**Review by jkschulz — APPROVED** [review]

File: src/write_gcm.c (hunk 1 above)
Comment (igorpeshansky) [reply src/write_gcm.c]: Good suggestion, thanks. Not sure what you meant by "bring the existing one down to this block" — I think the initial comment was actually supposed to cover both blocks, which is what I did. PTAL.

Comment (jkschulz) [reply src/write_gcm.c]: love it

**Review by jkschulz — APPROVED** [review]
perfection

**Pull request merged**